tensorboard/plugins/histogram/summary_test.py

```diff
@@ -316,14 +316,15 @@ def write_histogram_event(self, *args, **kwargs):
         with tf.name_scope("_") as temp_scope:
             scope = temp_scope.rstrip("/_")
 
-        @tf2.function
+        @tf2.function(autograph=True)
         def graph_fn():
             # Recreate the active scope inside the defun since it won't propagate.
             with tf.name_scope(scope):
                 self.call_histogram_op(*args, **kwargs)
 
         writer = tf2.summary.create_file_writer(self.get_temp_dir())
         with writer.as_default():
+            # print(tf2.autograph.to_code(graph_fn.python_function))
```

Comment:
Unintentional?  Also the `autograph=True` addition above seems like it shouldn't be necessary?  (I believe it's true by default.)